modules/core/src/v2/coins/abstractUtxoCoin.ts

```diff
@@ -233,6 +245,34 @@ export abstract class AbstractUtxoCoin extends BaseCoin {
     return this._network;
   }
 
+  static blockExplorer: UTXOBlockExplorer = {
+    baseUrl: "https://api.blockchair.com",
+
+    async getAddressUnspent (addressBase58: string): Bluebird<UnspentInfo[]> {
+      // make a call to blockchair apis
+      const url = `${AbstractUtxoCoin.blockExplorer.baseUrl}/${this.getFullName()}/outputs?addresses=${addressBase58}`;
```

Comment:
Blockchair's api is designed such that all UTXO chains can call this URL, and they just have to substitute their chain names in.

Reply:
Note that blockchair does not have BTG. (Yet another reason we should remove BTG. ;))